src/odc/core/Table.cc

```diff
@@ -397,7 +397,9 @@ Span Table::decodeSpan(const std::vector<std::string>& columns) {
     // Do the decoding
 
     std::vector<size_t> lastDecoded(ncols, 0);
-    double decodeBuffer[maxDoublesDecode];
+
+    std::vector<double> decodeBufferVec(maxDoublesDecode);
```

Comment:
Is this really a change we want to make. This shifts having the decode buffer from being on the stack to in the heap.